apps/dtsse/dtsse-dashboard-ingestion/dtsse-dashboard-ingestion.yaml

```diff
@@ -16,6 +16,6 @@ spec:
       chart: ./stable/dtsse-dashboard-ingestion
       sourceRef:
         kind: GitRepository
-        name: hmcts-charts
+        name: dtsse-dashboard-ingestion-dev
         namespace: flux-system
```

Comment:
Don't think this will work. The path is different and we need to keep the existing git repo in prod.